packages/traveling-fastlane/actions/manage_ad_hoc_provisioning_profile.rb

```diff
@@ -43,7 +43,31 @@ def register_missing_devices(udids)
 end
 
 def find_profile_by_bundle_id(bundle_id)
-  Spaceship::Portal.provisioning_profile.ad_hoc.find_by_bundle_id(bundle_id: bundle_id).first
+  expo_profiles = Spaceship::Portal.provisioning_profile.ad_hoc.find_by_bundle_id(bundle_id: bundle_id)
+  expo_profiles = expo_profiles.select{ |profile|
+    profile.name.start_with?('*[expo]') and profile.expires.to_datetime > DateTime.now
```

Comment:
https://github.com/expo/expo-cli/blob/master/packages/traveling-fastlane/actions/manage_ad_hoc_provisioning_profile.rb#L79
I think we could use `profile.valid?` instead of `profile.expires.to_datetime > DateTime.now`.

Reply:
i originally had `profile.valid?` but found out that validity is determined on:
- expiry date of profile
- validity of certificates associated with the profile

If the expo managed profile has invalid certs, but is still before the expiration date, we should use the profile and update it with our valid cert.